src/tools/schema2kotlin.ts

```diff
@@ -117,7 +117,7 @@ ${imports.join('\n')}
       if (type.isEntity || type.isVariable) {
           const node = type.isEntity
             ? nodes.find(n => n.variableName === null && n.schema.equals(type.getEntitySchema()))
-            : nodes.find(n => n.variableName.includes((type as TypeVariable).variable.name));
+            : nodes.find(n => n.variableName && n.variableName.includes((type as TypeVariable).variable.name));
```

Comment:
This should really be `n.variableName === (type as TypeVariable).variable.name`, but I can make this change in the follow up PR.

Reply:
:+1: